src/js/modules/articles/reducer.js

```diff
@@ -25,7 +23,17 @@ const reducer = (state = { ...initialState }, action) => {
         ...state,
         isFetching: false,
         isFetched: true,
-        response: action.payload,
+        /* Placing state.articles as the initial value will for some reason
+         * inhibit components' updating with the articles state, which is why
+         * the spread operator is required.
+         */
+        articles: {
+          ...state.articles,
+          ...action.payload.reduce((acc, article) => {
```

Comment:
can you elaborate? I don't understand what I did wrong and what you want me to change it to. Operator precedence? Well, I want things in the action to override the current state.articles if keys are the same because those articles may be more updated.